Carthage/Checkouts/DBNetworkStack/Tests/DBNetworkStackTests/RetryNetworkserviceTest.swift

```diff
@@ -59,7 +59,7 @@ class RetryNetworkserviceTest: XCTestCase {
         }, onError: { _ in
             XCTFail()
         })
-        networkServiceMock.returnError(error: .unknownError, count: errorCount)
+        networkServiceMock.returnError(with: .unknownError, count: errorCount)
```

Comment:
I like return `return(successful: ...)` && `return(error: ...)`

But this API comes from DBNetworkStack. So this projects only uses it. We could bring this up in a new PR at DBNetworkStack.